django/contrib/gis/db/backends/oracle/operations.py

```diff
@@ -240,11 +240,10 @@ def spatial_ref_sys(self):
         from django.contrib.gis.db.backends.oracle.models import OracleSpatialRefSys
         return OracleSpatialRefSys
 
-    def modify_insert_params(self, placeholders, params):
+    def modify_insert_params(self, placeholder, params):
         """Drop out insert parameters for NULL placeholder. Needed for Oracle Spatial
-        backend due to #10888
+        backend due to #10888.
```

Comment:
I think the Oracle CI is unfortunately not running GIS tests so we'll have to test against it somehow.